client/controllers/AddToPlanningController.tsx

```diff
@@ -20,6 +21,24 @@ const DEFAULT_PLANNING_SCHEMA = {
 };
 
 export class AddToPlanningController {
+    $scope: any;
+    notify: typeof superdeskApi['ui']['notify'];
```

Comment:
It's not correct to specify this type. You should not make assumptions on what superdesk API is using under the hood as implementation. It should be possible to easily change it without breaking anything.